src/Utils.ts

```diff
@@ -317,3 +320,22 @@ function nonZeroDataBytes(data: Uint8Array): number {
 
     return counter;
 }
+
+export function suffixData(relayRequest: RelayRequest, chainId: number) {
```

Comment:
Why do we define this function here, if it's used by the client only?

Reply:
Because I tried to follow the same grouping strategy I guess, so I include it there, because its something related with signature flow.